### PR TITLE
Validate the secret recovery phrase during recovery

### PR DESCRIPTION
**Pull request by miladsoft**
blockcore-wallet

### CODE REVIEW

File: angular/src/app/services/wallet-manager.ts [hunk 1]
```diff
@@ -67,6 +67,11 @@ export class WalletManager {
         return (this.secure.get(this.activeWalletId) != null);
     }
 
+    validateMnemonic( mnemonic: string) {
+      return of(bip39.validateMnemonic(mnemonic)).pipe(
+        delay(1000)
```
Comment (sondreb) [inline angular/src/app/services/wallet-manager.ts:72]: Why is there a delay here? It makes the UI behave a bit weird as validation can take up to 1 second?

**Review by sondreb — CHANGES_REQUESTED** [review]
Thanks for the PR, good work!

File: angular/src/app/services/wallet-manager.ts (hunk 1 above)
Comment (miladsoft) [reply angular/src/app/services/wallet-manager.ts:72]: I put it to show to the user. I better clean it. Because I do not show an error to the user.

Comment (sondreb) [reply angular/src/app/services/wallet-manager.ts]: The input field becomes red with warning, we can add additional error message below the textbox later. This delay simply delays the showing of the red error.

**Review by sondreb — APPROVED** [review]
Thanks for PR!

File: angular/src/app/services/wallet-manager.ts (hunk 1 above)
Comment (miladsoft) [reply angular/src/app/services/wallet-manager.ts]: Yes . I picked it up. I do not think the error message is necessary. The same red line is enough

**Pull request merged**